esp/src/eclwatch/DFUSearchWidget.js

```diff
@@ -69,57 +51,6 @@ define([
         postCreate: function (args) {
             this.inherited(arguments);
             this.borderContainer = registry.byId(this.id + "BorderContainer");
-            //this.tabContainer = registry.byId(this.id + "TabContainer");
```

Comment:
Remind me, where is all this getting moved to?

Reply:
In this case I am simply deleting old and unused commented code.

None of the ECL Watch functionality is moving to @hpcc-js in this pull request.